xio-core/src/test/java/com/xjeffrose/xio/config/ConfigReloaderUnitTest.java

```diff
@@ -48,122 +65,198 @@ String createConfig(String value) throws IOException, FileNotFoundException {
 
     public void update(TrivialConfig oldValue, TrivialConfig newValue) {
       this.config = newValue;
+      fireUpdated();
     }
 
     public void fireUpdated() {}
   }
 
-  @Test
-  public void testWiring() throws Exception {
-    ScheduledExecutorService executor = new ScheduledThreadPoolExecutor(1);
-    ConfigReloader<TrivialConfig> reloader = new ConfigReloader<>(executor, TrivialConfig::new);
-    TrivialConfig config = reloader.init(createConfig("10"));
-    TrivialState state =
-        new TrivialState(config) {
-          @Override
-          public void fireUpdated() {
-            assertEquals(20, this.config.limit);
-            executor.shutdown();
-          }
-        };
-    assertEquals(10, config.limit);
-    reloader.start(state::update);
-
-    Thread.sleep(2000);
+  private static void addPath(String s) throws Exception {
+    File f = new File(s);
+    URI u = f.toURI();
+    URLClassLoader urlClassLoader = (URLClassLoader) ClassLoader.getSystemClassLoader();
+    Class<URLClassLoader> urlClass = URLClassLoader.class;
+    Method method = urlClass.getDeclaredMethod("addURL", new Class[] {URL.class});
```

Comment:
line 79 and 81 - array is unnecessary for varargs

Reply:
ok changed